mitosheet/src/components/taskpanes/Graph/GraphSidebarTabs.tsx

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Mito
+
+import React from 'react';
+import { classNames } from '../../../utils/classNames';
+import { GraphSidebarTab } from '../../../types';
+import MitoAPI from '../../../api';
+
+/* 
+    The tabs at the bottom of the graph sidebar that let you switch between
+    setup, style, and export
+*/
+function GraphSidebarTabs(
+    props: {
+        selectedTab: GraphSidebarTab, 
+        setSelectedGraphSidebarTab: React.Dispatch<React.SetStateAction<GraphSidebarTab>>
+        mitoAPI: MitoAPI;
+    }): JSX.Element {
+
+    const changeTab = (tab: GraphSidebarTab) => {
+        props.setSelectedGraphSidebarTab(tab)
+
+        // Send a log message saying that this tab has been clicked,
+        // and keeping the tab at the highest level
+        void props.mitoAPI.log('clicked_graph_' + tab + '_tab');
+    }
+
+    return (
+        <div className='graph-sidebar-tab-container'>
```

Comment:
Can you open an issue for making a general Tabs component for all the tabs we have?

Reply:
Opened: https://github.com/mito-ds/monorepo/issues/171